packages/core/src/App/Containers/RealAccountSignup/account-wizard.jsx

```diff
@@ -259,9 +266,13 @@ const AccountWizard = props => {
                 } else {
                     props.onFinishSuccess(response.new_account_real.currency.toLowerCase());
                 }
-
-                const { document_type, document_number } = { ...form_values() };
-                if (document_type && document_number) {
+                const { document_type, document_number, document_additional } = { ...form_values() };
+                if (document_additional) {
```

Comment:
no need for this check if you give the parameter a default value in `submitIDVData`

Reply:
Will remove the conditional check here